services/wallet/api.go

```diff
@@ -28,7 +28,7 @@ type API struct {
 // TODO(dshulyak) benchmark loading many transfers from database. We can avoid json unmarshal/marshal if we will
 // read header, tx and receipt as a raw json.
 func (api *API) GetTransfers(ctx context.Context, start, end *hexutil.Big) ([]TransferView, error) {
-	log.Debug("call to get transfers", "start", start, "end", end)
```

Comment:
Make sure this change gets reverted.

Reply:
The reason I used `Info` instead of `Debug` is that it was too noisy. There should be better way to filter these logs, but I'm not aware of it. Except of adding some custom string to all logs of interest, but that would cause almost the same issue.